README.md

```diff
@@ -104,3 +104,24 @@ tfsec_formats: sarif,csv
 The screenshot below demonstrates the comments that can be expected when using the action
 
 ![Example PR Comment](images/pr_commenter.png)
+
+
+## Run directly (and not in Docker)
+
+This will avoid, pulling and build a new docker image on each usage :
+
+```yaml
+name: tfsec-pr-commenter
+on:
+  pull_request:
+jobs:
+  tfsec:
+    name: tfsec PR commenter
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Clone repo
+        uses: actions/checkout@master
+      - name: tfsec
+        uses: aquasecurity/tfsec-pr-commenter-action/composite@v1.2.0
```

Comment:
is this version correct?

Reply:
I am also looking to merge this PR: https://github.com/aquasecurity/tfsec-pr-commenter-action/pull/92/files, making sure versions are aligned properly

Reply:
The version here refere to git tag in this repo so it's correct but outdated, you're right. 

Does I need to put the latest one ? and the release process will update it or something more generic like `vx.y.z` ?

Reply:
Yes I think we should update the version, seeing as the composite folder has not been introduced yet and will break if referenced in v1.2.0, or v1.3.1 for that matter. I will create a new release for this as v1.3.2, so we should point the docs to this new version.

Reply:
@giorod3 Sorry for the latency. [Here](https://github.com/aquasecurity/tfsec-pr-commenter-action/pull/91/commits/c8d8d8dac00c75da21490df49277412a085a7959) a new commit to handle this point.